cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricClient.java

```diff
@@ -660,4 +660,16 @@ public void enableSchedule(ScheduleId scheduleId) throws Exception {
                                               "schedules", scheduleId.getSchedule(), "enable");
     verifyResponse(HttpResponseStatus.OK, responder.getStatus(), "Enable schedule failed");
   }
+
+  public void deleteSchedule(ScheduleId scheduleId) throws Exception {
+    MockResponder responder = new MockResponder();
+    ApplicationId application = scheduleId.getParent();
+    String uri = String.format("%s/apps/%s/versions/%s/schedules/%s", getNamespacePath(application.getNamespace()),
+                               application.getVersion(), application.getApplication(), scheduleId.getSchedule());
```

Comment:
application.getApplication() should come before version

Reply:
👍